components/gitpod-protocol/src/public-api-converter.ts

```diff
@@ -314,4 +326,53 @@ export class PublicAPIConverter {
         }
         return WorkspacePhase_Phase.UNSPECIFIED;
     }
+
+    toOrganization(org: ProtocolOrganization): Organization {
+        const result = new Organization();
+        result.id = org.id;
+        result.name = org.name;
+        result.creationTime = Timestamp.fromDate(new Date(org.creationTime));
+        return result;
+    }
+
+    toOrganizationMember(member: OrgMemberInfo): OrganizationMember {
+        const result = new OrganizationMember();
+        result.userId = member.userId;
+        result.fullName = member.fullName;
+        result.primaryEmail = member.primaryEmail;
+        result.avatarUrl = member.avatarUrl;
+        result.role = this.toOrgMemberRole(member.role);
+        result.memberSince = Timestamp.fromDate(new Date(member.memberSince));
+        result.ownedByOrganization = member.ownedByOrganization;
+        return result;
+    }
+
+    toOrgMemberRole(role: OrgMemberRole): OrganizationRole {
+        switch (role) {
+            case "owner":
+                return OrganizationRole.OWNER;
+            case "member":
+                return OrganizationRole.MEMBER;
+            default:
+                throw new Error(`unknown org member role ${role}`);
```

Comment:
Sohuld it be ORGANIZATION_ROLE_UNSPECIFIED here?

We should also always use `ConnectError` if we throw something from gRPC apis.